Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/StateContainer.cs

```diff
@@ -51,9 +43,17 @@ public void Dispatch(Command command)
             });
         }
 
-        public void Initialize()
+        public void Initialize(VisualElement rootVisualElement)
         {
+            m_RootVisualElement = rootVisualElement;
```

Comment:
Maybe add a comment here explaining why we do this little dance?
E.g. "We need to use a root element for TrackSerializedObjectValue that is destroyed with the view. Using a root element from the settings window would not enable the tracking callback to be destroyed or garbage collected."

Reply:
Good idea - have added the comment in 5e872116617b56d20a930efac85404e519874a53